src/components/Form/Form.js

```diff
@@ -27,6 +27,7 @@ const Form = ({ handleAddTip, validateInputs }) => {
 
     return(
         <form>
+            <h3>Share Your Advice</h3>
```

Comment:
You added a header that acts as a label for our form. This is great...much more of a user friendly experience.